src/commands/kv/mod.rs

```diff
@@ -76,7 +76,7 @@ fn help(error_code: u16) -> &'static str {
         // legacy namespace errors
         10021 | 10035 | 10038 => "Consider moving this namespace",
         // cloudflare account errors
-        10017 | 10026 => "Check your account settings in the Cloudflare dashboard",
+        10017 | 10026 => "Workers KV is only available on the Workers Unlimited plan, please upgrade your account to use",
```

Comment:
`please upgrade your account to use` -> I feel like we need an easier call to action here, maybe a link?

Reply:
also i think i would eliminate `to use` but that's just me